src/goto-synthesizer/synthesizer_utils.h

```diff
@@ -9,6 +9,9 @@ Author: Qinheping Hu
 #ifndef CPROVER_GOTO_SYNTHESIZER_SYNTHESIZER_UTILS_H
 #define CPROVER_GOTO_SYNTHESIZER_SYNTHESIZER_UTILS_H
 
+#include <util/pointer_expr.h>
```

Comment:
It's not obvious why this include is newly needed?

Reply:
I added it for fixing some CI failures. Let me remove it now and see what CI says.